backend/scripts/airtable/dumpProjects.js

```diff
@@ -46,23 +47,26 @@ async function dumpData() {
     .select({
       view: "All Proposals",
     })
-    .firstPage();
+    .all();
   if (data.length == 0) return console.error("Run this script again");
-  data.reverse();
+  data.reverse(); // reverse because old proposals might be missing some info
 
   const grantCategoryJsonReverse = {};
   Object.keys(grantCategoryJson).forEach((key) => {
     grantCategoryJsonReverse[grantCategoryJson[key]] = key;
   });
 
-  const done = [];
+  const allProjects = await Project.find({});
+
+  const done = allProjects.map((project) => project.projectName);
+
+  const allSigners = await Signer.find({});
 
+  const signersDone = allSigners.map((signer) => signer.address);
   for (let project of data.map((x) => x.fields)) {
-    const admin = project["Wallet Address"];
+    const admin = ethers.utils.getAddress(project["Wallet Address"]);
```

Comment:
I'm not familiar as to why retrieving an Address from ethers.utils would make this variable be named `admin`.
This is a little confusing to me.

I.E. this has nothing to do with trust levels...?